app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class StaticPagesController < ApplicationController
```

Comment:
add comments for class and correct indendation for empty methods

Reply:
done